README.md

```diff
@@ -1,6 +1,12 @@
 # Universal Robots ROS2 Driver
+Universal Robots have become a dominant supplier of lightweight, robotic manipulators for industry, as well as for scientific research and education. 
+<center><img src="ur_robot_driver/doc/initial_setup_images/e-Series.jpg" alt="Universal Robot e-Series family" style="width: 80%;"/></center>
 
-Beta version of the ROS2 Universal Robots driver. Should be transferred to the Universal Robots org when ready.
+This is one of the very first ROS2 manipulator drivers. Some of the new features are enabled by ROS2 and includes decreased latency, improved security, and more flexibility regarding middleware configuration. Launch files to get started quickly and easily with MoveIt2 are included. 
+
+This driver is developed on top of [Universal_Robots_Client_Library](https://github.com/UniversalRobots/Universal_Robots_Client_Library) and support some key cobot functionalities like; pause at emergency stop, safeguard stop, automatic speed scaling to avoid violate the safety setting and manually speed scaling from the teach pendant. In addition the externalControl URCap makes it possible to include ROS2 behaviors in the robot program,
```

Comment:
```suggestion
This driver is developed on top of [Universal_Robots_Client_Library](https://github.com/UniversalRobots/Universal_Robots_Client_Library) and support some key cobot functionalities like; pause at emergency stop, safeguard stop, automatic speed scaling to avoid violate the safety setting and manually speed scaling from the teach pendant. In addition the externalControl URCap makes it possible to include ROS2 behaviors in the robot program.
```